docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+services:
+  arma3:
+    image: synixebrett/arma3server
+    container_name: arma3
+    ports:
+      - '2302:2302/udp'
+      - '2303:2303/udp'
+      - '2304:2304/udp'
+      - '2305:2305/udp'
+      - '2306:2306/udp'
+    volumes:
+      - './missions:/arma3/mpmissions'
+      - './configs:/arma3/configs'
+      - './mods:/arma3/mods'
+      - './servermods:/arma3/servermods'
+    environment:
+      - ARMA_CONFIG=main.cfg
+      - STEAM_USER=USERNAMENOTEMAILADDRESS
+      - STEAM_PASSWORD=PASSWORD
```

Comment:
These two could be changed to use `.env` file values instead, that's how my local dev compose file is set up at least 😅

Reply:
Ah that would be great aswell, could you post an excerpt of how that is setup?

Reply:
https://github.com/Dahlgren/Arma3Server/commit/20f18da9fbd04a4a6b44723c152e0ccd14a6932e